readme.md

```diff
@@ -10,17 +7,24 @@ Hi. This gem allows you to perform the following conversions:
 * A zipcode to a lat/lon
 * A zipcode to just a lat
 * A zipcode to just a lon
-* A zipcode to its GMT offset
-* A state to its GMT offset
 * A lat/lon pair to a region
-* A lat/lon pair to its GMT offset
 * A lat/lon pair to its zip code
 
-Area uses public domain data and does not rely on any external services (the internets). Usage is meant to be more lightweight than the Geocoder gem.
+Original area gem includes the following conversions:
+* A zipcode to its GMT offset
+* A state to its GMT offset
+* A lat/lon pair to its GMT offset
 
-## Installation
+The difference between the area gem and this gem is the zipcodes.csv file. This gem uses the geonames US file from http://download.geonames.org/export/zip/US.zip. Said file needs the 0, 3, 5, 6, 7, 8 and 11 columns deleted in order to work.
```

Comment:
only clarification: does `said file` refer to the original geonames file? eg: we removed columns 0, 3, 5, 6, 7, 8, and 11 from geonames to adapt to area? 

if this is the case, i might try wording it like this: 
_The difference between the original area gem and this forked gem is the zipcodes.csv file. This gem uses the geonames US file from http://download.geonames.org/export/zip/US.zip, which provides a more up-to-date list of US-based zip codes. To meet the needs of the area gem format, we have adapted the original geonames file by removing columns 0, 3, 5, 6, 7, 8 and 11. All the information itself has renamed the same._